march_hardware/src/IMotionCube.cpp

```diff
@@ -461,23 +461,23 @@ std::string IMotionCube::parseDetailedError(uint16 detailedError)
 
 bool IMotionCube::goToTargetState(IMotionCubeTargetState targetState)
 {
-  ROS_INFO("\tTry to go to '%s'", targetState.getDescription().c_str());
+  ROS_DEBUG("\tTry to go to '%s'", targetState.getDescription().c_str());
   while (!targetState.isReached(this->getStatusWord()))
   {
     this->setControlWord(targetState.getControlWord());
-    ROS_INFO_THROTTLE(0.5, "\tWaiting for '%s': %s", targetState.getDescription().c_str(),
+    ROS_DEBUG_THROTTLE(0.5, "\tWaiting for '%s': %s", targetState.getDescription().c_str(),
```

Comment:
Without this you have no indication that anything is happening if the exo gets stuck in this loop. Either keep it in info, or throw info messages/warnings/an error after ~2-10 seconds trying.